pom.xml

```diff
@@ -11,7 +11,7 @@
 	</parent>
 
 	<artifactId>spring-cloud-gcp</artifactId>
-	<version>1.0.0.BUILD-SNAPSHOT</version>
+	<version>1.0.0.RC1</version>
```

Comment:
Can we split this into two PRs so that's it easier to do a rollback later? First, remove the beta modules, and then we'll prepare for the RC1 release separately.